snc.sh

```diff
@@ -35,6 +35,16 @@ else
     fi
 fi
 
+function apply_bootstrap_etcd_hack() {
+        # This is needed for now due to etcd changes in 4.4:
+        # https://github.com/code-ready/snc/wiki/Workaround-for-4.4--etcd-operator-addition
```

Comment:
Better to mention https://github.com/openshift/cluster-etcd-operator/pull/279 since this was the actual change when this hack is added.